broker/test/test_broker/mocks/index.js

```diff
@@ -44,8 +44,20 @@ function init() {
   return exports;
 }
 
-function verify() {
+function verify(ignoreList = []) {
   /* jshint expr:true */
+  if(nock.pendingMocks().length > 0) {
+    if(nock.pendingMocks().length == ignoreList.length) {
+      let count = 0;
+      for(let i = 0; i < ignoreList.length; i++) {
+        if(nock.pendingMocks()[i] == ignoreList[i])
```

Comment:
Implementaion looks good. Only thing is, do we need a sort step before so that we are sure about the ordering of both lists?

Reply:
We are sending the list from the UT itself. So, this sorting thing does not make much sense as we will be making the list in the order of pending mocks only.